bip-tap.mediawiki

```diff
@@ -535,12 +534,15 @@ where:
 *** If this type isn't present, then a <code>split_commitment</code> MUST be present.
 ** <code>split_commitment_proof</code>: is used to permit the spending of an asset UTXO created as a result of an asset split. When an asset is split, the non-change UTXO commits to the location of all other splits within an MS-SMT tree. When spending a change UTXO resulting from an asset_split, a normal <code>asset_witness</code> isn't required, instead the owner of the change asset UTXO must prove that it holds a valid split which was authorized by the main transfer transaction.
 *** Outputs with the same <code>split_commitment</code> are said to share a single <code>asset_witness</code> as such outputs are the result of a new asset split. Therefore we only need a single witness and the resulting merkle-sum asset tree to verify a transfer.
-** <code>asset_witness</code>: is a serialized witness in an identical format as Bitcoin's Segwit witness field. This field can only be blank if <code>prev_asset_input</code> is blank.
+** <code>asset_witness</code>: is a serialized witness in an identical format as Bitcoin's Segwit witness field. 
+*** If <code>prev_asset_input</code> is all zeroes, the witness will be either
+**** empty, to indicate this asset does not support further issuance. In this case <code>asset_group_key</code> MUST NOT be present.
+**** non-empty, in case the witness will be interpreted according to [[../master/bip-0340.mediawiki#specification|BIP-341]] validation rules using the <code>asset_group_key</code> in place of the <code>asset_script_hash</code>. In this case <code>asset_group_key</code> MUST be present.
```

Comment:
Where is `asset_raw_key` revealed? Since we need that to be revealed to verify the two levels of mapping.

Reply:
Also I think we need to distinguish from case of an initial mint (`asset_raw_key` being revealed for the first time), and subsequent ones (valid witness over `asset_group_key` and the virtual tx revealed).

Reply:
`asset_raw_group_key` is only revealed in the proof file (first mint), and is never needed again. 

After you verify the initial mint, you know that the `asset_group_key` is valid for the given `asset_id` and there is no need to distinguish initial mint from subsequent mints: 
They both need to have the virtual tx being signed with the `asset_group_key`. The only difference is the initial mint needs to prove the asset id matches the asset genesis.